### ISSUE TITLE
Build directory output for catch-all routes

### ISSUE THREAD

**Issue by Timer** [zeit/next.js#10190]
Thanks for the quick response to this. Unfortunately I don't think this is fully resolved:

On 9.2.1-canary.5,

With `exportTrailingSlash: true` in `next.config.js`,

With template `pages/[...slug].jsx`,
```
export async function unstable_getStaticPaths() {
	return [{ params: { slug: ['en_CA', 'about'] } }];
}

export async function unstable_getStaticProps({ params }) {
	console.log('\nparams', params);
	const { slug } = params;
	const props = {
		props: {
			slug,
		},
	};

	return props;
}
```

The build works, but the nested directories created are not as (I) expected:

Note the directory named `en_CA%2Fabout`:
![image](https://user-images.githubusercontent.com/10197537/72813074-bb7f4b80-3c30-11ea-8b86-482d216628d6.png)

This is the build console output. Note the slug is passed to `getStaticProps` as a single string within an array (`params { slug: [ 'en_CA/about' ] }`):

<img width="620" alt="image" src="https://user-images.githubusercontent.com/10197537/72813448-5546f880-3c31-11ea-9c04-65fe6f605fb6.png">

What I think is expected behavior - let me know if I'm off here:
* The directory structure should be created to match the the catch-all path
* The slug value is passed either as an array of strings (as returned from `getStaticPaths`) OR as a single string (concatenated with `/`), but not a single string within an array.

_Originally posted by @dpfavand in https://github.com/zeit/next.js/issues/10173#issuecomment-576715876_

**Comment by jamesvidler** [comment]
I can confirm this works when exportTrailingSlash is set to `false`.

**Comment by Timer** [comment]
Fixed via #10328 10379

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
